### PR TITLE
Fix React warnings in src/app/components/Metadata/index.test.jsx

### LINKED ISSUE #2777: Fix React warnings in src/app/components/Metadata/index.test.jsx
**Describe the bug**
There are React warnings in unit tests.

**To Reproduce**

1. Go to simorgh in terminal
2. Run npm run test:unit
3. Observe React warnings:

```
    console.log src/testHelpers/setupTests.js:19
      Test failed because too many React warnings were detected. Please fix the following:
      Warning: Failed prop type: Invalid prop `facebookAppID` of type `string` supplied to `Metadata`, expected `number`.
          in Metadata
    console.error src/testHelpers/setupTests.js:30
      Warning: Failed prop type: Invalid prop `facebookAppID` of type `string` supplied to `Metadata`, expected `number`.
          in Metadata
    console.error src/testHelpers/setupTests.js:30
      Warning: Failed prop type: Invalid prop `locale` of type `array` supplied to `Metadata`, expected `string`.
          in Metadata
    console.log src/testHelpers/setupTests.js:19
      Test failed because too many React warnings were detected. Please fix the following:
      Warning: Failed prop type: Invalid prop `themeColor` of type `number` supplied to `Metadata`, expected `string`.
          in Metadata
    console.error src/testHelpers/setupTests.js:30
      Warning: Failed prop type: Invalid prop `themeColor` of type `number` supplied to `Metadata`, expected `string`.
          in Metadata
    console.error src/testHelpers/setupTests.js:30
      Warning: Failed prop type: Invalid prop `type` of type `boolean` supplied to `Metadata`, expected `string`.
          in Metadata
    console.log src/testHelpers/setupTests.js:19
      Test failed because too many React warnings were detected. Please fix the following:
      Warning: Failed prop type: The prop `showArticleTags` is marked as required in `Metadata`, but its value is `undefined`.
          in Metadata
    console.error src/testHelpers/setupTests.js:30
      Warning: Failed prop type: The prop `showArticleTags` is marked as required in `Metadata`, but its value is `undefined`.
          in Metadata
```

**Expected behaviour**

No React warning for test suite.

**Testing notes**

No testing required.

### PR DESCRIPTION
**Pull request by FK78**
Resolves #2777

**Overall change:** _Fix warnings_

**Code changes:**

- _Add apostrophes._
- _Add missing IDs_
- _Correct order of props_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/app/components/ErrorPage/__snapshots__/index.test.jsx.snap [hunk 1]
```diff
@@ -293,11 +292,10 @@ exports[`ErrorMain should correctly render for an error page for Persian 1`] = `
   font-size: 2rem;
   line-height: 2.625rem;
   color: #3F3F42;
-  font-family: NassimPersian,Arial,Verdana,Geneva,Helvetica,sans-serif;
-  font-weight: 700;
-  font-style: normal;
+  font-family: ReithSerif,Helvetica,Arial,sans-serif;
```
Comment (dr3) [inline src/app/components/ErrorPage/__snapshots__/index.test.jsx.snap]: Hmm, something is causing the persian error page to now use 'news' as the service name, not 'persian' in this test

**Review by dr3 — CHANGES_REQUESTED** [review]
Getting a fair amount of snapshots failures, maybe needs a `npm ci && npm run updateSnapshots` :)

**Review by thekp — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Comment by dr3** [comment]
Only changes test files, doesnt need manual testing, Good to merge :)

**Pull request merged**